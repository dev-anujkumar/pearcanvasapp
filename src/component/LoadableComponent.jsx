import React, { Component } from 'react';

class LoadableComponent extends Component {
    constructor(props) {
        super(props);
        this.state = {}
    }

    render() {
        return (
            <React.Fragment>
                <h2>Your Lazy Component Is Loaded ....</h2>
            </React.Fragment>
        );
    }
}
export default LoadableComponent;