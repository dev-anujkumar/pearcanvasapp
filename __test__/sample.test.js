import React from 'react';
import LoadableComponent from '../src/component/LoadableComponent.jsx';

describe('testing tooltip component with case 1 props', () => {

    it('render tooltip component with class tooltipabove..', () => {
        const component = mount(<LoadableComponent />);
        console.log(component.debug());
        expect(component).toMatchSnapshot();
    })
})