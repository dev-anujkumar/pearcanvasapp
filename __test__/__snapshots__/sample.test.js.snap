// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing tooltip component with case 1 props render tooltip component with class tooltipabove.. 1`] = `
<LoadableComponent>
  <h2>
    Your Lazy Component Is Loaded ....
  </h2>
</LoadableComponent>
`;
